### DISCUSSION TITLE
Mark file as someone else's contribution

### DISCUSSION THREAD

**Discussion by Liebranca** [community/community#23367] [New to GitHub]
<p>Hello,</p>
<p>I have some files in a repository (small images stored through LFS) that I want to delete and replace with something more tightly compressed to go easier on the bandwidth. It’s far from a problem *right now* (usage is way below 10% per month) but I’d rather not let it -become- a problem later on.</p>
<p>Now, the issue is some of these files are not originally made by me and I’d rather keep them as the original author’s contribution rather than my own.</p>
<p>So, how do we go about that?</p>
<p>I can of course just reach out to the original author and ask them to upload the modified version but that would take longer. Is there a way to simply mark a file as ‘made by X’?</p>
<p>Thanks.</p>

**Comment by shrink** [comment, marked as answer]
<p>A commit is assigned to an author based on the commit metadata, therefore you <em>could</em> create a commit for another user by changing the metadata. However, it’s generally bad practice to create commits on behalf of someone else. Fortunately, GitHub supports the concept of “co-authors” which are displayed alongside the actual commit author – therefore, you can commit the changes yourself <em>and</em> identify that the content came from another user.</p>
<p>If you’re using the GitHub website to create commits then you can add the co-author trailer to your commit extended description, e.g:</p>
<pre><code class="lang-auto">Co-authored-by: Alice &lt;alice@example.com&gt;
</code></pre>
<p>A full outline of how to achieve this (on GitHub Desktop, via the command line and via <a href="http://github.com" rel="noopener nofollow ugc">github.com</a>) is available in the documentation, <a href="https://docs.github.com/en/github/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors" rel="noopener nofollow ugc">Creating a commit with multiple authors</a>:</p>
<blockquote>
<p>You can attribute a commit to more than one author by adding one or more <code>Co-authored-by</code> trailers to the commit’s message. Co-authored commits are visible on GitHub.</p>
</blockquote>
